src/azure/cli/utils/_msrestazure_threading.py

```diff
@@ -0,0 +1,24 @@
+#---------------------------------------------------------------------------------------------
```

Comment:
Yes. I created an issue in their repo: [https://github.com/Azure/autorest/issues/1378](https://github.com/Azure/autorest/issues/1378) 

The current implementation does not expose the ability to create daemon threads. This would work around the issue until such time that they (maybe) make this the default or configurable.

Another solution would be great too, but just trying to stop the running threads on KeyboardInterrupt failed. 

---
In reply to: [76327054](https://github.com/Azure/azure-cli/pull/757#discussion_r76327054) [](ancestors = 76327054)